charts/linkerd2/requirements.lock

```diff
@@ -12,4 +12,4 @@ dependencies:
   repository: file://../add-ons/tracing
   version: 0.1.0
 digest: sha256:d2428770ae7d5134c5af6521c78a4c5f95da4c75f21bdea0f74fad6ab6e2e044
-generated: "2020-06-24T11:07:53.924602129Z"
+generated: "2020-07-23T14:47:05.936935407+02:00"
```

Comment:
As `requirements.yaml` did not change in this PR, wonder if this is relevant/needed? :thinking: